src/main/kotlin/no/nav/k9/los/nyoppgavestyring/mottak/oppgave/OppgaveV3Repository.kt

```diff
@@ -251,39 +224,49 @@ class OppgaveV3Repository(
                     oppgavefelt = oppgavetype.oppgavefelter.first { oppgavefelt ->
                         oppgavefelt.id == row.long("oppgavefelt_id")
                     },
-                    verdi = row.string("verdi")
+                    verdi = row.string("verdi"),
+                    aktiv = row.boolean("aktiv"),
+                    oppgavestatus = row.stringOrNull("oppgavestatus")?.let {
+                        Oppgavestatus.fraKode(it)
+                    }
+
                 )
             }.asList
         )
     }
 
+    @VisibleForTesting
     fun lagreFeltverdier(
         oppgaveId: Long,
-        oppgaveFeltverdier: List<OppgaveFeltverdi>,
+        oppgave: OppgaveV3,
         tx: TransactionalSession
     ) {
         tx.batchPreparedNamedStatement("""
-            insert into oppgavefelt_verdi(oppgave_id, oppgavefelt_id, verdi)
-                    VALUES (:oppgaveId, :oppgavefeltId, :verdi)
+            insert into oppgavefelt_verdi(oppgave_id, oppgavefelt_id, verdi, aktiv, oppgavestatus)
+                    VALUES (:oppgaveId, :oppgavefeltId, :verdi, :aktiv, :oppgavestatus)
         """.trimIndent(),
-            oppgaveFeltverdier.map { feltverdi ->
+            oppgave.felter.map { feltverdi ->
                 mapOf(
                     "oppgaveId" to oppgaveId,
                     "oppgavefeltId" to feltverdi.oppgavefelt.id,
-                    "verdi" to feltverdi.verdi
+                    "verdi" to feltverdi.verdi,
+                    "aktiv" to oppgave.aktiv,
```

Comment:
Mulig å få inkonsistens her? Felt for aktiv i oppgavefeltverdi hentes fra input-objektet oppgave, mens det for oppgaven alltid lagres true